kong/plugins/ldap-auth/asn1.lua

```diff
@@ -127,7 +127,9 @@ _M.ASN1Decoder = {
     while (sPos < len) do
       local newSeq
       sPos, newSeq = self:decode(sStr, sPos)
-      if (not(newSeq) and self.stoponerror) then break end
+      if (not(newSeq) and self.stoponerror) then
```

Comment:
This file should be a vendor file, and not included in the plugins namespace :/ Too late now.

I forgot this rule in the contributing guide: no parenthesis in expressions, especially when they don't help clarifying said expression as is the case here.